js/tabs/js/lumx.tabs_directive.js

```diff
@@ -107,11 +270,11 @@ angular.module('lumx.tabs', [])
                 direction = 'left';
             }
 
-            var tabsWidth = links.outerWidth(),
+            var tabsVisibleWidth = links.parent('.tabs').outerWidth(),
                 activeTab = links.find('.tabs-link').eq($scope.activeTab),
```

Comment:
Is there a reason why the variables defined in the controller are not used for `.tabs` and `.tabs-link`? The active tab is dynamic, so it might be normal, but not the other one though.